src/frontend/next/src/components/GitHubContributorCard.tsx

```diff
@@ -0,0 +1,68 @@
+import { makeStyles } from '@material-ui/core/styles';
+import { Card, CardContent, Typography, Avatar, Grid } from '@material-ui/core';
+import useTelescopeContributor from '../hooks/use-telescope-contributor';
+
+function contributionsCount(contributions: number) {
+  return contributions < 3 ? 'contributions.' : `${contributions} contributions.`;
+}
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
You can specify theme to be of type "Theme". This fits with the typescript theme of types

Reply:
@HyperTHD : I don't think we need it because MUI has it inferred behind the scene. Thats why you don't see TS complains even without explicitly including it.

Reply:
Agree, see https://material-ui.com/guides/typescript/